docs/data/joy/components/drawer/drawer.md

```diff
@@ -7,64 +7,68 @@ githubLabel: 'component: drawer'
-
-The Drawer can be cancelled by clicking the overlay or pressing the Esc key.
-It closes when an item is selected, handled by controlling the `open` prop.
+The Drawer can be closed by clicking on the overlay or pressing the <kbd>Esc</kbd> key.
+It will also close if an item inside of it is selected, which can be controlled by the `open` prop.
 
 {{"demo": "DrawerBasic.js"}}
 
 ## Customization
 
 ### Anchor
 
-You can use the `anchor` prop for specifying where the drawer should appear from.
+Use the anchor prop to specify where the drawer should appear from.
+The default value is `left`.
 
 {{"demo": "DrawerAnchor.js"}}
 
-### Close button
+### Size
 
-Use the ModalClose component to add a close button to the drawer.
+Use the size prop to adjust the Drawer's width and height when positioned left/right and top/bottom, respectively.
 
-{{"demo": "DrawerCloseButton.js"}}
+{{"demo": "DrawerSize.js"}}
 
-### Size
+### Close button
 
-The `size` prop allows you to adjust the size of the drawer.
+Use the Modal Close component to add a close button to the Drawer.
+It automatically handles the `onClick` event.
 
-{{"demo": "DrawerSize.js"}}
+{{"demo": "DrawerCloseButton.js"}}
 
 ### Transition
 
-Set these CSS variables to the `sx` prop to the transition of the drawer:
+Customize the Drawer's opening transition by using the CSS variables below inside the sx prop:
 
-- `--Drawer-transitionFunction`: the [transition function](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function), default is `ease`.
-- `--Drawer-transitionDuration`: the [duration of the transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration), default is `0.3s`.
+- `--Drawer-transitionFunction`: the [transition function](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function); default is `ease`.
+- `--Drawer-transitionDuration`: the [duration of the transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration); default is `0.3s`.
 
 {{"demo": "DrawerTransition.js"}}
 
 ### Scrollable content
 
-Use the DialogContent component to create a scrollable content inside the drawer.
+Use the Dialog Content component to create a scrollable container inside the Drawer.
 
 {{"demo": "DrawerScrollable.js"}}
 
 ## Common examples
 
 ### Mobile navigation
 
+A common use case for the Drawer is mobile menus.
+
 {{"demo": "DrawerMobileNavigation.js"}}
 
-### Filter drawer
+### Inset drawer
 
-To create an inset panel, set the background and padding of the Drawer's content slot and then create a full height wrapper using [Sheet](/joy-ui/react-sheet/) or Box component.
+To make the Drawer not glued to the edge of the viewport, creating an inset style, use a combination of background and padding values on the Drawer component and either the Box or [Sheet](/joy-ui/react-sheet/) components for the full-height wrapper.
```

Comment:
Another tricky sentence—in this case I think it's best to extract the description of an inset drawer from the steps necessary to build one.

```suggestion
An inset drawer is a panel that's not anchored to any side of the screen.
You can achieve this by applying background and padding values to the Drawer component, and using either the Box or [Sheet](/joy-ui/react-sheet/) components for a full-height wrapper, as shown below:
```